lapin/src/config.rs

```diff
@@ -51,6 +46,29 @@ pub struct Config {
     pub connection_properties: lapin::ConnectionProperties,
 }
 
+pub(crate) struct ConnProps<'a>(pub(crate) &'a lapin::ConnectionProperties);
+impl<'a> std::fmt::Debug for ConnProps<'a> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("ConnectionProperties")
+            .field("locale", &self.0.locale)
+            .field("client_properties", &self.0.client_properties)
+            .finish_non_exhaustive()
+    }
+}
+
+impl std::fmt::Debug for Config {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Config")
+            .field("url", &self.url)
+            .field("pool", &self.pool)
+            .field(
+                "connection_properties",
+                &ConnProps(&self.connection_properties),
+            )
+            .finish()
+    }
+}
+
```

Comment:
`lapin::ConnectionProperties` no longer provides a Debug impl, but the fields are public, so I've implemented a wrapper for the fields that can be debugged so we don't lose any functionality